File: CallUrl.java
import java.net.InetAddress;

public class CallUrl{
    
    public static void main(String[] args)throws Exception{
        
        String urlString = null;

        String hostName = InetAddress.getLocalHost().getHostName();

        if(isInnerHost(hostName)){
            urlString = "http://inner.host";
        }else{
            urlString = "http://outter.host";
        }
    }

    private static boolean isInnerHost(String hostName){
        
        return true;
    }
}

File: src/main/scala/CallUrl.scala
import java.net._

def isInnerHost(hostName: String) = true

var urlString: String = null
var hostName = InetAddress.getLocalHost.getHostName
if(isInnerHost(hostName)){
  urlString = "http://inner.host"
}else{
  urlString = "http://outter.host"
}
